.travis.yml

```diff
@@ -4,6 +4,8 @@ rvm:
   - 2.2.3
 install:
   - bundle install
+  - nvm install 5.0
+  - nvm use 5.0
   - npm install -g npm
```

Comment:
@justin808 @mamute Since we are now install Node 5.0 with nvm, we can probably remove `npm install -g npm`. Node 5.0 comes with NPM 3.x.x.

Before travis was using a older version of npm that was failing our build.